collections/security/roles/google_authenticator/README.md

```diff
@@ -0,0 +1,25 @@
+# google_authenticator ( 2fa )
+
+## Description
+
+This role simply install google-authenticator tool and configure the basics to enable 2fa over ssh.
+
+## Instructions
+
+For now the Role has two possible configuration modes:
+
+1 - public key + google authenticator ( recomended )
+
+```yaml
+mfa_mode: password
```

Comment:
Shouldnt it be the opposite? So pubkey here ?

Reply:
yesss